pull-request.sh

```diff
@@ -254,7 +262,20 @@ main () {
                                 "${ASSIGNEES}" "${REVIEWERS}" "${TEAM_REVIEWERS}"
 
         fi
+    fi
+}
 
+# Run curl with default values
+safe_curl() {
+    printf "curl -fsSL -H 'AUTH...' %s\n" "$*" >&2
+    curl -fsSL -H "${AUTH_HEADER}" -H "${HEADER}" "$@"
+}
+
+# Print the response and, if FAIL_ON_ERROR, exit with an error
+curl_die() {
```

Comment:
This name is a little unsettling, haha, and it doesn't reflect what the function does. Perhaps something like `check_response` instead?

Reply:
how about `abort_if_fail`?

Reply:
sounds good!